tags/faq/origins.ytag

```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+For more specific help with the Origins mod, go to the dedicated Origins discord server: https://discord.gg/4mTMHu3
```

Comment:
Possibly a little terse? maybe `please go to` or `you can go to` would be better.